src/main/java/bio/terra/service/tabulardata/google/BigQuerySqlGenerator.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.service.tabulardata.google;
```

Comment:
I like the templating a lot! I'm not seeing a lot of value to factoring this out into a separate module, though.

My idea was to rearrange the Pdao by operations (dataset, snapshot, ingest) with a shared "utils" module. FWIW